lib/module.js

```diff
@@ -88,12 +88,36 @@ function setupHooks(options) {
   })
 }
 
-function extendBuild(config) {
-  const urlLoader = config.module.rules.find(rule => String(rule.test).includes('svg'))
+function extendBuild(config, options) {
+  const urlLoader = config.module.rules.find(rule => {
```

Comment:
I think we don't need to find `svg` rule, all we need is excluding output directory from all rules

```js
// exclude output directory from all the loaders
config.module.rules.forEach(rule => {
  if (!rule.exclude) rule.exclude = []
  rule.exclude.push(path.resolve(options.output))
})
```

And add our custom rule
```js
config.module.rules.push({
    test: /\.svg$/,
    include: [
      path.resolve(options.output)
    ],
    use: [{
      loader: 'file-loader'
    }]
  })
```

Reply:
also please remove `package-lock.json` from PR

Reply:
How do I include your changes into my PR?